applications/ShapeOptimizationApplication/python_scripts/mapper_factory.py

```diff
@@ -29,7 +29,17 @@ def CreateMapper(origin_model_part, destination_model_part, mapper_settings):
         "integration_method"         : "gauss_integration",
         "number_of_gauss_points"     : 5,
         "in_plane_morphing"                   : false,
-        "in_plane_morphing_settings"          : {}
+        "in_plane_morphing_settings"          : {},
+        "plane_symmetry"             : false,
+        "plane_symmetry_settings"    : {
+            "point" : [0.0, 0.0, 0.0],
+            "normal": [1.0, 0.0, 0.0]
```

Comment:
Wouldnt it makes more sense to leave every entry zero and then, during validation of the settings, make sure a reasonable value is specified? I mean, even if one wants to use those default settings it would be better to directly see them in the application settings rather an assuming the user knows this somewhat arbitrary default behavior.